.github/workflows/unit-tests.yml

```diff
@@ -0,0 +1,49 @@
+name: "phpunit tests"
```

Comment:
To save time later when we add phpcs & whatnot, let's title this "PHP tests" and name the file `php-tests.yml`.